lib/puppet/face/help/action.erb

```diff
@@ -1,4 +1,7 @@
 <%# encoding: UTF-8%>
+<% if action.deprecated? -%>
+<%= "Warning: 'puppet #{action.name}' is deprecated and will be removed in a future release." %>
```

Comment:
Should this be `puppet #{action.face.name} #{action.name}` so it's clear which face's action is deprecated?

Reply:
yep. good call.